docs/implementation-references/proxy-settings.md

```diff
@@ -2,6 +2,126 @@
 
 This document lists the different proxy settings that need to be set. 
 
+## Script to automate proxy setup.
```

Comment:
Is this our internal script?

Reply:
Yes. Removed all internal proxy details.